docs/user/quick-start.md

```diff
@@ -130,15 +133,26 @@ There are two ways to deploy TeleIRC persistently:
 
 This section explains how to configure and install TeleIRC as a simple executable binary.
 
-#### Requirements
+**NOTE**:
+**This assumes you are building from source.**
+If you use a pre-built binary from a [GitHub Release][14], skip to [_Configuration_](#configuration).
+
+#### Pre-requirements
 
 - git
 - go (v1.13 and v1.14 supported)
 
-#### Install dependencies
+Packages for these pre-requirements are available on most `*NIX` distributions.
+Check your distribution documentation for more info on how to install these packages.
+
+#### Build TeleIRC
 
-1. Clone the repository (`git clone https://github.com/RITlug/teleirc.git`)
+This section is only required if you are building a binary from source:
+
+1. Clone repository (`git clone https://github.com/RITlug/teleirc.git`)
+1. Enter repository (`cd teleirc/`)
 1. Install dependencies (`go install`)
+1. Build binary (`go build ./...`)
```

Comment:
I feel this should be `go build cmd/teleirc.go` as _this_ is what will ultimately create an executable `teleirc` binary in the current working directory.

Reply:
@Tjzabel Fixed in commit c78d50314c83e5e51e6e77fd764576a54bd8b976, thanks for catching that.